contracts/chain-adapters/L1_Adapter.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "./Base_Adapter.sol";
+import "../interfaces/AdapterInterface.sol";
+import "../interfaces/WETH9.sol";
+
+import "@uma/core/contracts/common/implementation/Lockable.sol";
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract L1_Adapter is Base_Adapter, Lockable {
```

Comment:
@mrice32 @chrismaree I originally added this because I wanted to test `HubPool.relaySpokePoolAdminFunction()` in the simplest way and check that it can delegate a call to another contract, so I figured why not just build this L1 adapter to kill two birds with one stone